README.md

```diff
@@ -92,17 +92,17 @@ The JSON response contains an array of datapoints. Each datapoint contains the f
 
 ### Submission
 
-Create a new repo into your favorite git platform (GitHub, Gitlab, etc) with a README file containing list of instructions to run the project.
+Create a new repo into your favorite git platform (GitHub, Gitlab, etc) with a README file containing a list of instructions to run the project. Bonus points if you deploy your solution.
```

Comment:
I'm unsure if this might create extended pressure to get the Bonus points and work longer on it for a deployment? Together with the potential design challenge I think this challenge is already big enough. What is your opinion on that concern?